omod/src/main/webapp/pages/results.gsp

```diff
@@ -156,7 +217,9 @@ def prettify = { "$it".replaceAll("null", "").replaceAll("\\s*,\\s", " ") }
         patients.push("${prettify(ui.encodeHtmlContent(it.name))}");
     </script>
     <tr>
-        <td style="display: none">rule_${it.rule.ruleId}</td>
+        <td class="hidden">rule_${it.rule.ruleId}</td>
+        <td class="hidden">${it.patient.patientIdentifier}</td>
```

Comment:
@ssmusoke do you mean to say we should display `program id` if its present otherwise `patient preferred id` ? If that's the case please let me know how can I get it - as result data is generated from another module's code. Maybe we should fetch patient's PreferredIdentifier while saving results.